### PR TITLE
[GITHUB-58] Support Authorization logs

### LINKED ISSUE #58: Cant see authorization denied logs
As a platform alert system
I would like to be informed when an authorization is denied
So that I can detect users with authorization issues

### PR DESCRIPTION
**Pull request by mhqdotio**
Resolves #58

### CODE REVIEW

File: molecule/default/converge.yml [hunk 1]
```diff
@@ -8,6 +8,16 @@
     sansible_kafka_environment_vars:
       - "NEWRELIC_OPTS=\"-javaagent:/home/{{ sansible_kafka_user }}/newrelic/newrelic.jar\""
       - "export TEST_KAFKA_OPTS=\"${KAFKA_OPTS} ${NEWRELIC_OPTS}\""
+    sansible_kafka_server_properties:
+      listeners: "PLAINTEXT://127.0.0.1:9092"
+      broker.id: 11
+
+  pre_tasks:
+    - name: Install
```
Comment (wojtek-oledzki) [inline molecule/default/converge.yml]: Move to "prepare.yaml"

**Review by wojtek-oledzki — APPROVED** [review]

**Comment by lobsterdore** [comment]
Tests are broken at the moment

**Comment by wojtek-oledzki** [comment]
@lobsterdore it's green!

supergreen

**Review by lobsterdore — APPROVED** [review]
LGTM 👍

**Pull request merged**